### ISSUE TITLE
Missing list of Golang versions supported in the documentation.

### ISSUE THREAD

**Issue by dmigwi** [google/gops#88]
I just noticed that the documentation does not have list golang versions supported by this package.  It cannot run using other golang versions apart from `go1.12`

```shell
go build github.com/google/gops/signal: module requires Go 1.12
go build github.com/google/gops/internal: module requires Go 1.12
``

**Comment by rayunstop** [comment]
any update?

**Comment by rakyll** [comment]
We are actually not testing this versions earlier than 1.12. If it is critical, I may consider adding 1.11 as well. But Go has a policy to maintain only the last two versions so I expect everyone to update to 1.12 at least.

**Comment by tklauser** [comment]
I think this can be closed, given that Go 1.12 was release more than 3 years ago and the most recent versions supported by the Go project are 1.17 and 1.18 at this point.
